src/components/organisms/wizard/Wizard.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+
+import React, { useState } from "react";
+import { Button } from "~/components/atoms/Button";
+import FormInput from "../form/formInput/FormInput";
+import { useForm } from "react-hook-form";
+import FormComponent from "../form/formComponent/FormComponent";
+
+interface InputConfig {
+  type: string;
+  placeholder: string;
+  label: string;
+}
+
+interface WizardProps {
+  steps: string[];
+  inputs: InputConfig[][];
+}
+
+const Wizard: React.FC<WizardProps> = ({ steps, inputs }) => {
+  const [currentStep, setCurrentStep] = useState(0);
+  const [formData, setFormData] = useState<Record<string, string>>({});
+
+  const nextStep = () => {
+    if (currentStep < steps.length - 1) {
+      setCurrentStep(currentStep + 1);
+    }
+  };
+
+  const prevStep = () => {
+    if (currentStep > 0) {
+      setCurrentStep(currentStep - 1);
+    }
+  };
+
+  const handleChange = (
+    e: React.ChangeEvent<HTMLInputElement>,
+    index: number,
+  ) => {
+    setFormData({
+      ...formData,
+      [`${currentStep}-${index}`]: e.target.value,
+    });
```

Comment:
```suggestion
    setFormData({
      ...formData,
      [e.target.name]: e.target.value,
    });
```
Using [`${currentStep}-${index}\`] will make it super hard for us to send it to the BE later. Either way, we will use react-hook-form to submit the forms but it's something to keep in mind :).